docs/index.html

```diff
@@ -204,6 +236,50 @@ <h2>#TwitterThrowback</h2>
 <img id="shareImage" src="./img/twitter_icon.png"></button>
 <button onclick="backToTop()" id="topBtn" title="Back to top">👆</button>
 
+<script>
+    // Set the default date input value to today
+    var month = document.getElementById('month');
```

Comment:
My convention is to use `let` or `const`. If these variables will be referenced globally, please use `window.` so it's clear. Also, reference them like that across the codebase so it's consistent.

Reply:
Corrected. All 3 bindings are now defined with the window object